matches/src/main/java/com/renovavision/scorebat/matches/details/MatchDetailsFragment.kt

```diff
@@ -0,0 +1,54 @@
+package com.renovavision.scorebat.matches.details
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.renovavision.scorebat.matches.R
+import com.renovavision.scorebat.matches.databinding.FragmentMatchDetailsBinding
+import com.renovavision.scorebat.matches.list.MatchesAdapter
+import com.renovavision.scorebat.network.model.Match
+import com.renovavision.scorebat.utils.bindingDelegate
+import com.renovavision.scorebat.utils.onViewLifecycle
+import javax.inject.Inject
+
+class MatchDetailsFragment @Inject constructor() : Fragment() {
+
+    private val binding by bindingDelegate(FragmentMatchDetailsBinding::bind)
+
+    private lateinit var matchDetailsAdapter: MatchDetailsAdapter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        matchDetailsAdapter = MatchDetailsAdapter()
```

Comment:
@AlexGolovach You can use `private val matchDetailsAdapter = MatchDetailsAdapter()` instead of `lateinit`